### ISSUE TITLE
Fail to train of GAIL in Ant-v2 environment

### ISSUE THREAD

**Issue by seolhokim** [Khrylx/PyTorch-RL#28]
I trained your ppo first.

` python examples/ppo_gym.py --env-name Ant-v2 --save-model-interval 100 `

After 500 episodes, I made trajectories.

` python gail/save_expert_traj.py --model-path assets/learned_models/Ant-v2_ppo.p `

Last, I ran gail.

` python gail/gail_gym.py --env-name Ant-v2 --expert-traj-path assets/expert_traj/Ant-v2_expert_traj.p `

I implemented Gail and Vail, but I failed to train it too.(but hopper worked well)

Any Ideas?

**Comment by dibbla** [comment]
Hi! Have you work it out?
It seems to be the problem of zfilter.

**Comment by seolhokim** [comment]
No. I guess you doubt the training has failed because of standardization by zfilter, right? I checked but it was not the key to solving problem in my implementation.

**Comment by lviano** [comment]
Any solution to this ? I am running in the same problem. The code works well for all the other MuJoCo I have tried but not for HalfCheetah
